packages/tools/client-debugger/client-debugger-view/src/components/ContainerSelectionDropdown.tsx

```diff
@@ -0,0 +1,80 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { Dropdown, IDropdownOption, IDropdownStyles, IStackTokens, Stack } from "@fluentui/react";
+import React from "react";
+
+import { IFluidClientDebugger } from "@fluid-tools/client-debugger";
+import { HasClientDebuggers, HasContainerId } from "../CommonProps";
+
+/**
+ * {@link ContainerSelectionDropdownProps} input props.
+ *
+ * @internal
+ */
+export interface ContainerSelectionDropdownProps extends HasClientDebuggers, HasContainerId {
+    /**
+     * Take the selected container id to set as current viewed container id.
+     * @param containerId - current selected container id.
+     */
+    onChangeSelection(containerId: string): void;
+}
+
+/**
+ * A dropdown that displays all registered containers.
+ *
+ */
+export function ContainerSelectionDropdown(
+    props: ContainerSelectionDropdownProps,
+): React.ReactElement {
+    const dropdownStyles: Partial<IDropdownStyles> = {
+        dropdown: { width: "300px", zIndex: "1" },
+    };
+
+    const stackTokens: IStackTokens = { childrenGap: 20 };
+
+    const { clientDebuggers, containerId } = props;
+
+    let selectedKey = containerId;
+
+    function renewContainerOptions(debuggers: IFluidClientDebugger[]): IDropdownOption[] {
+        const options: IDropdownOption[] = [];
+        for (const x of debuggers) {
+            if (x.containerId === containerId) {
+                selectedKey = x.containerNickname ?? x.containerId;
+            }
+            options.push({
+                key: x.containerId,
+                text: x.containerNickname ?? x.containerId,
+            });
+        }
+        return options;
+    }
+
+    const clientDebuggerOptions = renewContainerOptions(clientDebuggers);
+
+    const _onClientDebuggerDropdownChange = (
+        event: React.FormEvent<HTMLDivElement>,
+        option?: IDropdownOption,
+    ): void => {
+        if (option) {
+            const selectedDebugger = clientDebuggers.find((c) => {
```

Comment:
Nit: for accessibility reasons, I always recommend against single-letter variables like `c`. I would recommend `debugger` or `clientDebugger` or something more descriptive instead :)

Reply:
Renamed it!

Reply:
Thanks!